docs/websites.mdx

```diff
@@ -0,0 +1,133 @@
+---
+description: 'Hosting and managing websites with Nitric'
+---
+
+# Websites
+
+Nitric provides a simple way to deploy websites to the cloud, allowing you to serve website assets in a serverless environment without managing complex infrastructure.
+
+If you're interested in the architecture, provisioning, or deployment steps, they can be found [here](/architecture/websites).
+
+<Note>
+  At this time, websites can only be deployed as static assets. If you require
+  more advanced features, let us know:
+  https://github.com/nitrictech/nitric/issues
+</Note>
+
+## Overview
+
+Nitric’s website resource lets you:
+
+- Serve static files (HTML, JS, CSS, images) from cloud storage.
+- Service your entire nitric application from a single domain.
+- APIs are automatically served under `/api` and websites are served under `/`.
+- Deploy to various supported providers with minimal configuration.
+- Use infrastructure from code to keep your environment consistent.
+
+## Enabling Websites
+
+Websites are currently in [Preview](/reference/preview-features). To enable this feature in your project add the following to your `nitric.yaml` file.
+
+```yaml
+preview:
+  - websites
+```
+
+## Create a Website
+
+### 1. Create your website project
+
+Create a new website project using your preferred framework. For example, using [Astro](https://astro.build/) within your nitric project:
+
+```bash
+npm create astro@latest my-website
+```
+
+### 2. Add a website resources
+
+Add a website resource to your `nitric.yaml` file.
+
+```yaml
+websites:
+  - basedir: ./my-website
+    build:
+      command: npm run build
+      output: ./dist
+    dev:
+      command: npm run dev -- --port 4322
+      url: http://localhost:4322
+```
+
+### 3. Start your website locally
+
+Run `nitric start` to start your website locally. This will use the `dev` configuration, which proxies requests to your local development server.
+
+Run `nitric run` command to run your production website locally. This will use the `build` configuration and serve the static files.
+
+### 4. Deploy your website
+
+Run `nitric up` to deploy your website to the cloud.
+
+## Sub Sites
+
+You can also configure sub-sites under different paths. These will be deployed as subdirectories of the main site.
+
+Here is an example configuration:
+
+```yaml
+websites:
+  - basedir: ./my-website
+    build:
+      command: npm run build
+      output: ./dist
+    dev:
+      command: npm run dev -- --port 4322
+      url: http://localhost:4322
+  # the sub site is served under /docs, ensure you do not have conflicting routes
```

Comment:
I think this note about conflicting routes would be useful as a sub-heading under this section of the docs, with a little more detail about what happens if overlaps occur.